diff -up haxe-3.0/Makefile.x86_64 haxe-3.0/Makefile
--- haxe-3.0/Makefile.x86_64	2013-09-10 09:18:19.780792398 +0900
+++ haxe-3.0/Makefile	2013-09-10 09:24:15.199779694 +0900
@@ -11,6 +11,7 @@
 .SUFFIXES : .ml .mli .cmo .cmi .cmx .mll .mly
 
 INSTALL_DIR=/usr
+LIBDIR = $(INSTALL_DIR)/lib
 
 OUTPUT=haxe
 EXTENSION=
@@ -63,12 +64,12 @@ tools: haxelib haxedoc
 
 install:
 	cp haxe $(INSTALL_DIR)/bin/haxe
-	rm -rf $(INSTALL_DIR)/lib/haxe/std
-	-mkdir -p $(INSTALL_DIR)/lib/haxe
-	svn export std/ $(INSTALL_DIR)/lib/haxe/std
-	-mkdir -p $(INSTALL_DIR)/lib/haxe/lib
-	chmod -R a+rx $(INSTALL_DIR)/lib/haxe
-	chmod 777 $(INSTALL_DIR)/lib/haxe/lib
+	rm -rf $(LIBDIR)/haxe/std
+	-mkdir -p $(LIBDIR)/haxe
+	svn export std/ $(LIBDIR)/haxe/std
+	-mkdir -p $(LIBDIR)/haxe/lib
+	chmod -R a+rx $(LIBDIR)/haxe
+	chmod 777 $(LIBDIR)/haxe/lib
 	cp std/tools/haxelib/haxelib.sh $(INSTALL_DIR)/bin/haxelib
 	cp std/tools/haxedoc/haxedoc.sh $(INSTALL_DIR)/bin/haxedoc
 	chmod a+rx $(INSTALL_DIR)/bin/haxe $(INSTALL_DIR)/bin/haxelib $(INSTALL_DIR)/bin/haxedoc
