diff -up v4.03-9411/src/makefiles/linux_64bit.mak.destdir v4.03-9411/src/makefiles/linux_64bit.mak
--- v4.03-9411/src/makefiles/linux_64bit.mak.destdir	2014-01-07 07:56:51.117045999 +0900
+++ v4.03-9411/src/makefiles/linux_64bit.mak	2014-01-07 11:11:14.454321575 +0900
@@ -33,10 +33,10 @@ OPTIONS_COMPILE_RELEASE=-DNDEBUG -DVPN_S
 OPTIONS_LINK_RELEASE=-O2 -fsigned-char -m64 -lm -ldl -lrt -lpthread -lssl -lcrypto -lreadline -lncurses -lz
 
 INSTALL_BINDIR=/usr/bin/
-INSTALL_VPNSERVER_DIR=/usr/vpnserver/
-INSTALL_VPNBRIDGE_DIR=/usr/vpnbridge/
-INSTALL_VPNCLIENT_DIR=/usr/vpnclient/
-INSTALL_VPNCMD_DIR=/usr/vpncmd/
+INSTALL_VPNSERVER_DIR=/usr/lib64/softether/vpnserver/
+INSTALL_VPNBRIDGE_DIR=/usr/lib64/softether/vpnbridge/
+INSTALL_VPNCLIENT_DIR=/usr/lib64/softether/vpnclient/
+INSTALL_VPNCMD_DIR=/usr/lib64/softether/vpncmd/
 
 ifeq ($(DEBUG),YES)
 	OPTIONS_COMPILE=$(OPTIONS_COMPILE_DEBUG)
@@ -377,6 +377,8 @@ bin/vpncmd/hamcore.se2: src/bin/BuiltHam
 tmp/objs/vpncmd.o: src/vpncmd/vpncmd.c $(HEADERS_MAYAQUA) $(HEADERS_CEDAR) $(OBJECTS_MAYAQUA) $(OBJECTS_CEDAR)
 	$(CC) $(OPTIONS_COMPILE) -c src/vpncmd/vpncmd.c -o tmp/objs/vpncmd.o
 
+DESTDIR =
+
 # Install
 install: $(INSTALL_BINDIR)vpnserver $(INSTALL_BINDIR)vpnbridge $(INSTALL_BINDIR)vpnclient $(INSTALL_BINDIR)vpncmd
 	@echo
@@ -391,40 +393,44 @@ install: $(INSTALL_BINDIR)vpnserver $(IN
 	@echo
 
 $(INSTALL_BINDIR)vpnserver: bin/vpnserver/hamcore.se2 bin/vpnserver/vpnserver
-	@mkdir -p $(INSTALL_VPNSERVER_DIR)
-	cp bin/vpnserver/hamcore.se2 $(INSTALL_VPNSERVER_DIR)hamcore.se2
-	cp bin/vpnserver/vpnserver $(INSTALL_VPNSERVER_DIR)vpnserver
-	echo "#!/bin/sh" > $(INSTALL_BINDIR)vpnserver
-	echo $(INSTALL_VPNSERVER_DIR)vpnserver '"$$@"' >> $(INSTALL_BINDIR)vpnserver
-	echo 'exit $$?' >> $(INSTALL_BINDIR)vpnserver
-	chmod 755 $(INSTALL_BINDIR)vpnserver
+	@mkdir -p $(DESTDIR)/$(INSTALL_BINDIR)
+	@mkdir -p $(DESTDIR)/$(INSTALL_VPNSERVER_DIR)
+	cp bin/vpnserver/hamcore.se2 $(DESTDIR)/$(INSTALL_VPNSERVER_DIR)hamcore.se2
+	cp bin/vpnserver/vpnserver $(DESTDIR)/$(INSTALL_VPNSERVER_DIR)vpnserver
+	echo "#!/bin/sh" > $(DESTDIR)/$(INSTALL_BINDIR)vpnserver
+	echo $(INSTALL_VPNSERVER_DIR)vpnserver '"$$@"' >> $(DESTDIR)/$(INSTALL_BINDIR)vpnserver
+	echo 'exit $$?' >> $(DESTDIR)/$(INSTALL_BINDIR)vpnserver
+	chmod 755 $(DESTDIR)/$(INSTALL_BINDIR)vpnserver
 
 $(INSTALL_BINDIR)vpnbridge: bin/vpnbridge/hamcore.se2 bin/vpnbridge/vpnbridge
-	@mkdir -p $(INSTALL_VPNBRIDGE_DIR)
-	cp bin/vpnbridge/hamcore.se2 $(INSTALL_VPNBRIDGE_DIR)hamcore.se2
-	cp bin/vpnbridge/vpnbridge $(INSTALL_VPNBRIDGE_DIR)vpnbridge
-	echo "#!/bin/sh" > $(INSTALL_BINDIR)vpnbridge
-	echo $(INSTALL_VPNBRIDGE_DIR)vpnbridge '"$$@"' >> $(INSTALL_BINDIR)vpnbridge
-	echo 'exit $$?' >> $(INSTALL_BINDIR)vpnbridge
-	chmod 755 $(INSTALL_BINDIR)vpnbridge
+	@mkdir -p $(DESTDIR)/$(INSTALL_BINDIR)
+	@mkdir -p $(DESTDIR)/$(INSTALL_VPNBRIDGE_DIR)
+	cp bin/vpnbridge/hamcore.se2 $(DESTDIR)/$(INSTALL_VPNBRIDGE_DIR)hamcore.se2
+	cp bin/vpnbridge/vpnbridge $(DESTDIR)/$(INSTALL_VPNBRIDGE_DIR)vpnbridge
+	echo "#!/bin/sh" > $(DESTDIR)/$(INSTALL_BINDIR)vpnbridge
+	echo $(INSTALL_VPNBRIDGE_DIR)vpnbridge '"$$@"' >> $(DESTDIR)/$(INSTALL_BINDIR)vpnbridge
+	echo 'exit $$?' >> $(DESTDIR)/$(INSTALL_BINDIR)vpnbridge
+	chmod 755 $(DESTDIR)/$(INSTALL_BINDIR)vpnbridge
 
 $(INSTALL_BINDIR)vpnclient: bin/vpnclient/hamcore.se2 bin/vpnclient/vpnclient
-	@mkdir -p $(INSTALL_VPNCLIENT_DIR)
-	cp bin/vpnclient/hamcore.se2 $(INSTALL_VPNCLIENT_DIR)hamcore.se2
-	cp bin/vpnclient/vpnclient $(INSTALL_VPNCLIENT_DIR)vpnclient
-	echo "#!/bin/sh" > $(INSTALL_BINDIR)vpnclient
-	echo $(INSTALL_VPNCLIENT_DIR)vpnclient '"$$@"' >> $(INSTALL_BINDIR)vpnclient
-	echo 'exit $$?' >> $(INSTALL_BINDIR)vpnclient
-	chmod 755 $(INSTALL_BINDIR)vpnclient
+	@mkdir -p $(DESTDIR)/$(INSTALL_BINDIR)
+	@mkdir -p $(DESTDIR)/$(INSTALL_VPNCLIENT_DIR)
+	cp bin/vpnclient/hamcore.se2 $(DESTDIR)/$(INSTALL_VPNCLIENT_DIR)hamcore.se2
+	cp bin/vpnclient/vpnclient $(DESTDIR)/$(INSTALL_VPNCLIENT_DIR)vpnclient
+	echo "#!/bin/sh" > $(DESTDIR)/$(INSTALL_BINDIR)vpnclient
+	echo $(INSTALL_VPNCLIENT_DIR)vpnclient '"$$@"' >> $(DESTDIR)/$(INSTALL_BINDIR)vpnclient
+	echo 'exit $$?' >> $(DESTDIR)/$(INSTALL_BINDIR)vpnclient
+	chmod 755 $(DESTDIR)/$(INSTALL_BINDIR)vpnclient
 
 $(INSTALL_BINDIR)vpncmd: bin/vpncmd/hamcore.se2 bin/vpncmd/vpncmd
-	@mkdir -p $(INSTALL_VPNCMD_DIR)
-	cp bin/vpncmd/hamcore.se2 $(INSTALL_VPNCMD_DIR)hamcore.se2
-	cp bin/vpncmd/vpncmd $(INSTALL_VPNCMD_DIR)vpncmd
-	echo "#!/bin/sh" > $(INSTALL_BINDIR)vpncmd
-	echo $(INSTALL_VPNCMD_DIR)vpncmd '"$$@"' >> $(INSTALL_BINDIR)vpncmd
-	echo 'exit $$?' >> $(INSTALL_BINDIR)vpncmd
-	chmod 755 $(INSTALL_BINDIR)vpncmd
+	@mkdir -p $(DESTDIR)/$(INSTALL_BINDIR)
+	@mkdir -p $(DESTDIR)/$(INSTALL_VPNCMD_DIR)
+	cp bin/vpncmd/hamcore.se2 $(DESTDIR)/$(INSTALL_VPNCMD_DIR)hamcore.se2
+	cp bin/vpncmd/vpncmd $(DESTDIR)/$(INSTALL_VPNCMD_DIR)vpncmd
+	echo "#!/bin/sh" > $(DESTDIR)/$(INSTALL_BINDIR)vpncmd
+	echo $(INSTALL_VPNCMD_DIR)vpncmd '"$$@"' >> $(DESTDIR)/$(INSTALL_BINDIR)vpncmd
+	echo 'exit $$?' >> $(DESTDIR)/$(INSTALL_BINDIR)vpncmd
+	chmod 755 $(DESTDIR)/$(INSTALL_BINDIR)vpncmd
 
 # Clean
 clean:
