diff -up statsmodels-0.5.0/setup.py.disable_runtime_dep_check statsmodels-0.5.0/setup.py
--- statsmodels-0.5.0/setup.py.disable_runtime_dep_check	2014-01-13 04:18:39.019142501 +0900
+++ statsmodels-0.5.0/setup.py	2014-01-13 04:18:55.944143614 +0900
@@ -460,7 +460,7 @@ if __name__ == "__main__":
         # 3.3 needs numpy 1.7+
         min_versions.update({"numpy" : "1.7.0b2"})
 
-    check_dependency_versions(min_versions)
+    #check_dependency_versions(min_versions)
     write_version_py()
 
     # this adds *.csv and *.dta files in datasets folders
