diff -up xlsbeans-1.2.5/src/test/java/net/java/amateras/xlsbeans/XLSBeansTest.java.test xlsbeans-1.2.5/src/test/java/net/java/amateras/xlsbeans/XLSBeansTest.java
--- xlsbeans-1.2.5/src/test/java/net/java/amateras/xlsbeans/XLSBeansTest.java.test	2015-02-09 03:36:52.510475303 +0900
+++ xlsbeans-1.2.5/src/test/java/net/java/amateras/xlsbeans/XLSBeansTest.java	2015-02-09 03:51:35.631905395 +0900
@@ -11,6 +11,8 @@ import net.java.amateras.xlsbeans.xssfco
  * @author Naoki Takezoe
  */
 public class XLSBeansTest extends TestCase {
+	static String date_org = "06/04/10";
+	static String date_mod = "4/10/06";
 
 	public void test1() throws Exception {
 
@@ -26,7 +28,7 @@ public class XLSBeansTest extends TestCa
 		// DumpXLSBeans.showDump(userList);
 
 		assertEquals("User list2", userList.getTitle());
-		assertEquals("06/04/10", userList.getLastUpdate());
+		assertEquals(date_mod, userList.getLastUpdate());
 
 		assertEquals(new Point(2, 23), userList.getUnitUsers().get(0)
 				.getUnitUsers().get(0).getNamePosition());
@@ -105,7 +107,7 @@ public class XLSBeansTest extends TestCa
 		// DumpXLSBeans.showDump(userList);
 
 		assertEquals("User list2", userList.getTitle());
-		assertEquals("06/04/10", userList.getLastUpdate());
+		assertEquals(date_org, userList.getLastUpdate());
 
 		assertEquals(new Point(2, 23), userList.getUnitUsers().get(0)
 				.getUnitUsers().get(0).getNamePosition());
@@ -184,7 +186,7 @@ public class XLSBeansTest extends TestCa
 		// DumpXLSBeans.showDump(userList);
 
 		assertEquals("User list2", userList.getTitle());
-		assertEquals("06/04/10", userList.getLastUpdate());
+		assertEquals(date_org, userList.getLastUpdate());
 
 		assertEquals(new Point(2, 23), userList.getUnitUsers().get(0)
 				.getUnitUsers().get(0).getNamePosition());
@@ -266,7 +268,7 @@ public class XLSBeansTest extends TestCa
 		UserList userList = userLists[0];
 
 		assertEquals("User list2", userList.getTitle());
-		assertEquals("06/04/10", userList.getLastUpdate());
+		assertEquals(date_mod, userList.getLastUpdate());
 	}
 
 	public void test2_hssf() throws Exception {
@@ -288,7 +290,7 @@ public class XLSBeansTest extends TestCa
 		UserList userList = userLists[0];
 
 		assertEquals("User list2", userList.getTitle());
-		assertEquals("06/04/10", userList.getLastUpdate());
+		assertEquals(date_org, userList.getLastUpdate());
 	}
 
 	public void test2_xssf() throws Exception {
@@ -310,7 +312,7 @@ public class XLSBeansTest extends TestCa
 		UserList userList = userLists[0];
 
 		assertEquals("User list2", userList.getTitle());
-		assertEquals("06/04/10", userList.getLastUpdate());
+		assertEquals(date_org, userList.getLastUpdate());
 	}
 
 	public void test3() throws Exception {
@@ -593,7 +595,7 @@ public class XLSBeansTest extends TestCa
 		// DumpXLSBeans.showDump(userList);
 
 		assertEquals("User list2", userList.getTitle());
-		assertEquals("06/04/10", userList.getLastUpdate());
+		assertEquals(date_mod, userList.getLastUpdate());
 
 		assertEquals(new Point(2, 23), userList.getUnitUsers().get(0)
 				.getUnitUsers().get(0).getNamePosition());
