diff -up evilvte.git/src/config.h.custom evilvte.git/src/config.h
--- evilvte.git/src/config.h.custom	2013-07-09 00:01:11.900073763 +0900
+++ evilvte.git/src/config.h	2013-07-09 00:34:25.256188567 +0900
@@ -1,11 +1,11 @@
 /* Use // to disable options                                                  */
 
-#define    BACKGROUND_IMAGE       ".config/evilvte/background.png"
-// #define BACKGROUND_SATURATION  0.4
+// #define    BACKGROUND_IMAGE       ".config/evilvte/background.png"
+#define    BACKGROUND_SATURATION  0.5
 // #define BACKGROUND_SCROLLABLE  TRUE
-// #define BACKGROUND_TINT_COLOR  "black"
-// #define BACKGROUND_TRANSPARENT TRUE  /* Pseudo transparent background      */
-// #define BACKGROUND_OPACITY     TRUE  /* True transparent background        */
+#define    BACKGROUND_TINT_COLOR  "black"
+#define    BACKGROUND_TRANSPARENT TRUE  /* Pseudo transparent background      */
+#define    BACKGROUND_OPACITY     TRUE  /* True transparent background        */
 // #define BACKSPACE_KEY /* Options: AUTO, BACKSPACE, DELETE, ERASE_TTY,      */
                          /*          DELETE_SEQUENCE                          */
 // #define DELETE_KEY    /* Options: AUTO, BACKSPACE, DELETE, ERASE_TTY,      */
@@ -41,18 +41,18 @@
 // #define COLOR_TEXT_BOLD        "red"
 // #define COLOR_TEXT_DIM         "#FFFF00"
 // #define COLOR_TEXT_HIGHLIGHTED "green"
-// #define COMMAND_AT_ROOT_WINDOW TRUE  /* -r option, run in root window      */
-// #define COMMAND_COLOR_FG       TRUE  /* -fg option, foreground color       */
-// #define COMMAND_COLOR_BG       TRUE  /* -bg option, background color       */
-// #define COMMAND_SATURATION     TRUE  /* -sa option, saturation level       */
+#define COMMAND_AT_ROOT_WINDOW TRUE  /* -r option, run in root window      */
+#define COMMAND_COLOR_FG       TRUE  /* -fg option, foreground color       */
+#define COMMAND_COLOR_BG       TRUE  /* -bg option, background color       */
+#define COMMAND_SATURATION     TRUE  /* -sa option, saturation level       */
 // #define COMMAND_DOCK_MODE      TRUE  /* -d option, run as a dock           */
 #define    COMMAND_EXEC_PROGRAM   TRUE  /* -e option, execute program         */
-// #define COMMAND_FULLSCREEN     TRUE  /* -f option, init in fullscreen mode */
+#define COMMAND_FULLSCREEN     TRUE  /* -f option, init in fullscreen mode */
 // #define COMMAND_FONT           TRUE  /* -fn option, to assign font & size  */
 // #define COMMAND_GEOMETRY       TRUE  /* -g +X+Y option, to assign geometry */
-// #define COMMAND_LOGIN_SHELL    TRUE  /* -ls option, to enable login shell  */
+#define COMMAND_LOGIN_SHELL    TRUE  /* -ls option, to enable login shell  */
 // #define COMMAND_SET_TITLE      TRUE  /* -T or -title to set program title  */
-// #define COMMAND_SHOW_HELP      TRUE  /* -h option, show help               */
+#define    COMMAND_SHOW_HELP      TRUE  /* -h option, show help               */
 #define    COMMAND_SHOW_OPTIONS   TRUE  /* -o option, show build-time options */
 #define    COMMAND_SHOW_VERSION   TRUE  /* -v option, show program version    */
 // #define COMMAND_TAB_NUMBERS    TRUE  /* -[2-9] options, to set tab numbers */
@@ -77,15 +77,15 @@
 // #define SCROLL_LINES           1000  /* Negative value means unlimited     */
 // #define SCROLL_ON_KEYSTROKE    TRUE
 // #define SCROLL_ON_OUTPUT       TRUE
-#define    SCROLLBAR              RIGHT /* Options: LEFT, RIGHT, OFF_L, OFF_R */
+// #define SCROLLBAR              RIGHT /* Options: LEFT, RIGHT, OFF_L, OFF_R */
 // #define SHOW_WINDOW_BORDER     TRUE
-// #define SHOW_WINDOW_DECORATED  TRUE
+#define    SHOW_WINDOW_DECORATED  FALSE
 #define    SHOW_WINDOW_ICON       TRUE
-#define    STATUS_BAR             TRUE
+#define    STATUS_BAR             FALSE
 // #define WINDOW_TITLE_DYNAMIC   TRUE  /* for xterm escape sequences         */
 #define    WORD_CHARS             "-A-Za-z0-9_$.+!*(),;:@&=?/~#%"
 
-#define    MENU                   TRUE
+#define MENU                   TRUE
 // #define MENU_ENCODING_LIST     "BIG-5", "Default Encoding", "GBK", "EUC-JP"
 // #define MENU_MATCH_STRING_EXEC "firefox"
 // #define      MATCH_STRING_L    "firefox" /* left click to open without menu*/
@@ -93,7 +93,7 @@
 // #define      MATCH_STRING_HTTP TRUE      /* Detect http(s) and ftp(s)      */
 // #define      MATCH_STRING_MAIL TRUE      /* Detect mailto:                 */
 // #define      MATCH_STRING_FILE TRUE      /* Detect file:///                */
-// #define MENU_CUSTOM            "Copy", "Separator", "Paste", "Separator"
+#define MENU_CUSTOM            "Copy", "Separator", "Paste", "Separator", "Zoom in", "Separator", "Zoom out", "Separator", "Toggle fullscreen"
            /* Options: "Copy",         "Zoom in",       "Add tab",
             *          "Paste",        "Zoom out",      "Remove tab",
             *          "Select all",   "Zoom default",  "New window",
@@ -113,11 +113,12 @@
             *          "Auto-click" (when there is only one effective menu item)
             */
 
-#define    TAB                    TRUE
-// #define TAB_BORDER             0
+#define TAB                    TRUE
+#define TAB_BORDER             0
 // #define TAB_CLOSE_BUTTON       TRUE
 // #define TAB_EXPANDED_WIDTH     TRUE
 // #define TAB_LABEL              "Page %u" /* %u will be replaced by numbers */
+#define TAB_LABEL              "%u" /* %u will be replaced by numbers */
 // #define TAB_LABEL_DYNAMIC      TRUE      /* for xterm escape sequences     */
 // #define TAB_LABEL_CUSTOM       "One", "Two", "Three", "Four", "Five", "Six"
 // #define TAB_NEW_PATH_EQUAL_OLD TRUE
@@ -126,7 +127,7 @@
 // #define TAB_SHOW_INFO_AT_TITLE TRUE
 // #define TABBAR                 TRUE
 // #define TABBAR_PLACE           TOP    /* Options: TOP, BOTTOM, LEFT, RIGHT */
-// #define TABBAR_AUTOHIDE        TRUE
+#define TABBAR_AUTOHIDE        TRUE
 // #define TABBAR_SCROLLABLE      TRUE
 // #define TABBAR_MENU_SELECT_TAB TRUE
 
@@ -145,14 +146,14 @@
  *
  * See /usr/include/gtk-?.0/gdk/gdkkeysyms.h for GDK_* definitions.           */
 
-#define    HOTKEY                       TRUE
-// #define HOTKEY_COPY                  CTRL_SHIFT(GDK_C) || CTRL_SHIFT(GDK_c)
-// #define HOTKEY_PASTE                 CTRL_SHIFT(GDK_V) || CTRL_SHIFT(GDK_v)
+#define HOTKEY                       TRUE
+#define HOTKEY_COPY                  CTRL_SHIFT(GDK_C) || CTRL_SHIFT(GDK_c)
+#define HOTKEY_PASTE                 CTRL_SHIFT(GDK_V) || CTRL_SHIFT(GDK_v)
 // #define HOTKEY_SELECT_ALL            CTRL_SHIFT(GDK_S) || CTRL_SHIFT(GDK_s)
 // #define HOTKEY_COLOR_BACKGROUND      CTRL_SHIFT(GDK_B) || CTRL_SHIFT(GDK_b)
 // #define HOTKEY_EDIT_ENCODING         CTRL_SHIFT(GDK_U) || CTRL_SHIFT(GDK_u)
-// #define HOTKEY_FONT_BIGGER           CTRL(GDK_KP_Add)
-// #define HOTKEY_FONT_SMALLER          CTRL(GDK_KP_Subtract)
+#define HOTKEY_FONT_BIGGER           CTRL(GDK_KP_Add)
+#define HOTKEY_FONT_SMALLER          CTRL(GDK_KP_Subtract)
 // #define HOTKEY_FONT_DEFAULT_SIZE     CTRL(GDK_KP_Insert)
 // #define HOTKEY_FONT_SELECT           CTRL_SHIFT(GDK_F) || CTRL_SHIFT(GDK_f)
 // #define HOTKEY_MIMIC_SCROLL_UP       CTRL_SHIFT(GDK_K) || CTRL_SHIFT(GDK_k)
@@ -161,8 +162,8 @@
 // #define HOTKEY_RESET_TERMINAL        CTRL_SHIFT(GDK_R) || CTRL_SHIFT(GDK_r)
 // #define HOTKEY_RESET_AND_CLEAR       CTRL_SHIFT(GDK_A) || CTRL_SHIFT(GDK_a)
 // #define HOTKEY_SATURATION_DIALOG     CTRL_SHIFT(GDK_X) || CTRL_SHIFT(GDK_x)
-// #define HOTKEY_SATURATION_MORE       CTRL(GDK_KP_Multiply)
-// #define HOTKEY_SATURATION_LESS       CTRL(GDK_KP_Divide)
+#define HOTKEY_SATURATION_MORE       CTRL(GDK_KP_Multiply)
+#define HOTKEY_SATURATION_LESS       CTRL(GDK_KP_Divide)
 // #define HOTKEY_SCROLL_ONE_PAGE_UP    CTRL(GDK_k) || CTRL(GDK_K)
 // #define HOTKEY_SCROLL_ONE_PAGE_DOWN  CTRL(GDK_j) || CTRL(GDK_J)
 // #define HOTKEY_SEARCH_STRING         CTRL_SHIFT(GDK_Q) || CTRL_SHIFT(GDK_q)
@@ -180,7 +181,7 @@
 // #define HOTKEY_TAB_EDIT_LABEL        CTRL_SHIFT(GDK_E) || CTRL_SHIFT(GDK_e)
 // #define HOTKEY_TOGGLE_ANTI_ALIAS     CTRL_SHIFT(GDK_L) || CTRL_SHIFT(GDK_l)
 // #define HOTKEY_TOGGLE_DECORATED      CTRL(GDK_Left)
-// #define HOTKEY_TOGGLE_FULLSCREEN     CTRL_SHIFT(GDK_M) || CTRL_SHIFT(GDK_m)
+#define HOTKEY_TOGGLE_FULLSCREEN     CTRL_SHIFT(GDK_M) || CTRL_SHIFT(GDK_m)
 // #define HOTKEY_TOGGLE_HOTKEYS        CTRL(GDK_quoteleft)
 // #define HOTKEY_TOGGLE_ON_TOP         CTRL_SHIFT(GDK_O) || CTRL_SHIFT(GDK_o)
 // #define HOTKEY_TOGGLE_SCROLLBAR      CTRL(GDK_Right)
@@ -205,4 +206,4 @@
 // #define LABEL_MENU_TOGGLE_TABBAR     "_Toggle tabbar"
 // #define LABEL_SUBMENU_ENCODING       "_Character Encoding"
 // #define LABEL_SUBMENU_IME            "_Input Methods"
-// #define GTK3_CSS                     "GtkNotebook GtkHBox GtkButton { -GtkWidget-focus-line-width: 0; } GtkNotebook GtkButton { border-width: 0; padding: 0; -GtkButton-inner-border: 0; } GtkWindow GtkNotebook { padding: 0; }"
+#define GTK3_CSS                     "GtkNotebook GtkHBox GtkButton { -GtkWidget-focus-line-width: 0; } GtkNotebook GtkButton { border-width: 0; padding: 0; -GtkButton-inner-border: 0; } GtkWindow GtkNotebook { padding: 0; }"
diff -up evilvte.git/src/evilvte.c.custom evilvte.git/src/evilvte.c
